package.json

```diff
@@ -262,7 +262,7 @@
   "private": true,
   "dependencies": {
     "@babel/runtime": "^7.17.7",
-    "@department-of-veterans-affairs/component-library": "^10.2.2",
+    "@department-of-veterans-affairs/component-library": "^10.3.1-0",
```

Comment:
Once the fix is confirmed and https://github.com/department-of-veterans-affairs/component-library/pull/451 is merged, I will update this to use the full release.